static/app/views/issueDetails/groupSimilarIssues/similarStackTrace/index.tsx

```diff
@@ -49,15 +50,22 @@ function SimilarStackTrace({project}: Props) {
   const navigate = useNavigate();
   const prevLocationSearch = usePrevious(location.search);
   const hasSimilarityFeature = project.features.includes('similarity-view');
+  const {data: projectData, isPending} = useDetailedProject({
+    orgSlug: organization.slug,
+    projectSlug: project.slug,
+  });
   const hasSimilarityEmbeddingsFeature =
-    project.features.includes('similarity-embeddings') ||
+    projectData?.features.includes('similarity-embeddings') ||
```

Comment:
might add a  comment here about this flag only being available on project details